lib/device_api/android/adb.rb

```diff
@@ -232,28 +232,43 @@ def self.screencap( serial, args )
         
         shell(serial, cmd)
       end
+      
+      # Connects to remote android device
+      # @param [String] ip_address or ip_address:port
+      # @example
+      #  DeviceAPI::ADB.connect(ip_address, port)  
+      #  DeviceAPI::ADB.connect(ip_address_and_port)
+      def self.connect(ip_address, port=5555)
 
-      def self.connect(ipaddress, port = 5555)
-        ipaddressandport = "#{ipaddress}:#{port}"
-        check_ip_address(ipaddressandport)
-        cmd = "adb connect #{ipaddressandport}"
+        value = ip_address.split(":").last
```

Comment:
Could we push this to what ever is calling this method please? The ip_address parameter should be just that, if we have a named parameter there shouldn't be any ambiguity as to what is being passed.